installing/installing_ibm_powervs/installing-ibm-power-vs-customizations.adoc

```diff
@@ -0,0 +1,63 @@
+:_content-type: ASSEMBLY
+[id="installing-ibm-power-vs-customizations"]
+= Installing a cluster on {ibmpowerProductName} Virtual Server with customizations
+include::_attributes/common-attributes.adoc[]
+:context: installing-ibm-power-vs-customizations
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a customized cluster on infrastructure that the installation program provisions on {ibmpowerProductName} Virtual Server. To customize the installation, you modify parameters in the `install-config.yaml` file before you install the cluster.
+
+:FeatureName: {ibmpowerProductName} Virtual Server using installer-provisioned infrastructure
+include::snippets/technology-preview.adoc[]
+
+[id="prerequisites_installing-ibm-powervs-customizations"]
+== Prerequisites
+
+* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+* You read the documentation on xref:../../installing/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
+* You xref:../../installing/installing_ibm_powervs/installing-ibm-cloud-account-power-vs.adoc#installing-ibm-cloud-account-power-vs[configured an IBM Cloud account] to host the cluster.
+* If you use a firewall, you xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configured it to allow the sites] that your cluster requires access to.
+* You configured the `ccoctl` utility before you installed the cluster. For more information, see xref:../../installing/installing_ibm_powervs/preparing-to-install-on-ibm-power-vs.adoc#cco-ccoctl-configuring_preparing-to-install-on-ibm-power-vs[Configuring the Cloud Credential Operator utility].
```

Comment:
This prerequisite is different in the preview. Perhaps Netlify didn't update it? Can you check that this prereq is correct? 

![image](https://github.com/openshift/openshift-docs/assets/25209155/c38e24f6-d8d9-4991-b37f-06ead832224f)

Reply:
@mburke5678 yes , Netlify did not update the changes.

Reply:
It is apprearing like below : 
```
* You configured the `ccoctl` utility before you installed the cluster. For more information, see Configuring the Cloud Credential Operator utility.
* 
```
The code is pointing to the ID in the prerequisites assembly.

Code  : `* You configured the ccoctl utility before you installed the cluster. For more information, see xref:../../installing/installing_ibm_powervs/preparing-to-install-on-ibm-power-vs.adoc#choosing-an-method-to-install-ocp-on-power-vs-installer-provisioned[Configuring the Cloud Credential Operator utility].`

Reply:
@alishaIBM Should this link go directly to the Configuring the Cloud Credentials Operator module? 

`installing/installing_ibm_power/preparing-to-install-on-ibm-power-vs.adoc#cco-ccoctl-configuring_preparing-to-install-on-ibm-power-vs`

Reply:
Addressed here: https://github.com/openshift/openshift-docs/pull/60058/files#diff-379ae9fae203aa3c338c0ca5a56f806166e7c1d7eb29820bcc9875e3827980d2

Reply:
Few more updates w.r.to above are addressed in this PR : https://github.com/openshift/openshift-docs/pull/60086/files fyi.